demos/node-classification/gcn/gcn-cora-node-classification-example.ipynb

```diff
@@ -4,14 +4,36 @@
    "cell_type": "markdown",
```

Comment:
<p>I think this has lost the explanation of 'patience', which is useful?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
I tried a few different ways to phrase the explanation, and decided it was better to just link to the Keras docs and let them explain the parameters. That is, not try to teach too much about Keras here too. What do you think?

Reply:
I see, yes that works fine.